packages/app/package.json

```diff
@@ -4,38 +4,55 @@
   "private": true,
   "bundled": true,
   "dependencies": {
-    "@backstage/app-defaults": "^0.1.1",
-    "@backstage/catalog-model": "^0.9.6",
-    "@backstage/cli": "^0.10.0",
-    "@backstage/core-app-api": "^0.1.19",
-    "@backstage/core-components": "^0.7.2",
-    "@backstage/core-plugin-api": "^0.2.2",
-    "@backstage/integration-react": "^0.1.13",
-    "@backstage/plugin-api-docs": "^0.6.12",
-    "@backstage/plugin-catalog": "^0.7.2",
-    "@backstage/plugin-catalog-import": "^0.7.3",
-    "@backstage/plugin-catalog-react": "^0.6.2",
-    "@backstage/plugin-github-actions": "^0.4.22",
-    "@backstage/plugin-jenkins": "^0.5.11",
-    "@backstage/plugin-org": "^0.3.27",
-    "@backstage/plugin-scaffolder": "^0.11.9",
-    "@backstage/plugin-search": "^0.5.0",
-    "@backstage/plugin-tech-radar": "^0.4.11",
-    "@backstage/plugin-techdocs": "^0.12.4",
-    "@backstage/plugin-user-settings": "^0.3.10",
-    "@backstage/test-utils": "^0.1.20",
-    "@backstage/theme": "^0.2.12",
+    "@backstage/app-defaults": "^0.2.0",
+    "@backstage/catalog-model": "^0.12.0",
+    "@backstage/cli": "^0.15.0",
+    "@backstage/core-app-api": "^0.6.0",
+    "@backstage/core-components": "^0.9.0",
+    "@backstage/core-plugin-api": "^0.8.0",
+    "@backstage/integration-react": "^0.1.24",
+    "@backstage/plugin-api-docs": "^0.8.1",
+    "@backstage/plugin-azure-devops": "^0.1.17",
+    "@backstage/plugin-badges": "^0.2.25",
+    "@backstage/plugin-catalog": "^0.9.1",
+    "@backstage/plugin-catalog-common": "^0.2.0",
+    "@backstage/plugin-catalog-graph": "^0.2.13",
+    "@backstage/plugin-catalog-import": "^0.8.4",
+    "@backstage/plugin-catalog-react": "^0.8.0",
+    "@backstage/plugin-circleci": "^0.3.1",
+    "@backstage/plugin-cloudbuild": "^0.3.1",
+    "@backstage/plugin-code-coverage": "^0.1.28",
+    "@backstage/plugin-github-actions": "^0.5.1",
+    "@backstage/plugin-jenkins": "^0.7.0",
+    "@backstage/plugin-kubernetes": "^0.6.1",
+    "@backstage/plugin-lighthouse": "^0.3.1",
+    "@backstage/plugin-newrelic-dashboard": "^0.1.9",
```

Comment:
can we remove some of the plugins here we don't need, things like circle ci, cloudbuild, azuredevops, newrelic etc?